sky/skylet/providers/aws/config.py

```diff
@@ -215,7 +223,7 @@ def print_info(
     cli_logger.newline()
 
 
-def bootstrap_aws(config):
+def bootstrap_aws(config, skypilot_iam_role: bool = False):
```

Comment:
actually I would just set `skypilot_iam_role` as a global variable and set it to true -- as it is indeed always true in skypilot. we can do this because this is a feature flag other than an actual option

Reply:
We can make it global, but we need to change the variable and modify the `node_provider.py`, as the previous cluster previously launched by the skypilot need to use the old AWSNodeProvider to avoid backward compatibility issue. Wdyt?

Reply:
hmmm, I did not realize it is not compatible with previous solutions. then I am ok to keep it as a parameter. in the future we may perform migration for old permission and deprecate the old scripts.